### ISSUE TITLE
Option to render value directly on Input field

### ISSUE THREAD

**Issue by robhunt3r** [selectize/selectize.js#664]
I think we need (if it doesn't yet exists) an option to force use a single Input field instead of rendering an `span` or other custom content...

I will explain with an example (without code, I'm out home)

I'm trying to render an input with bootstrap3 at NavBar (typical search form) I needed to modify less files to make it working properly as it was absolutely messed up (only using with NavBar) but there's a problem that can't be fixed, or better say, it's a hell... When we select/add an option, selectize renders that element with our custom html code or default html code, `span` or `div` or whatever... 
That's OK if we are selecting some elements like tags, emails or whatever, but when we just need to perform a search and use an autocomplete with selectizejs (by autocomplete I mean an ajax request) we don't want a rendered block with our value, we just want TEXT, because we can delete a char without deleting the entire tag (even using `restore_on_backspace` is ugly) because it doesn't delete the first char on first DEL press, it deletes the rendered block and then it deletes the first char on the second DEL press..

Another buggy thing is that input (which is `position: absolute;`) is rendered again next to our value block, causing an overwidth on our final form...

So basically, what I mean (because I can't explain better without screenshots or code XD) is add an option to prevent block rendering and use only the `absolute` input field and treat our value as it is, a simple text...

What do you think? I think it's needed in some cases, maybe I am too exquisite.

**Comment by brianreavis** [comment]
I'm not following 100%, but it sounds like you're looking for something other than selectize. Maybe implementing something from scratch could work? If it's helpful, there's always [sifter.js](https://github.com/brianreavis/sifter.js) to take care of the option search aspect of it
